lib/cim/index.ts

```diff
@@ -0,0 +1,3 @@
+export {CimAdapter} from "./cim-adapter";
+export {IriProvider} from "./iri-provider";
+export {PrefixIriProvider} from "./prefix-iri-provider";
```

Comment:
I think in the repository we use empty line at the end of the file, does this code pass the ```npm run lint``` command? If so I may need to update the rules.